lib/WorkflowCommons.groovy

```diff
@@ -73,6 +73,53 @@ class WorkflowCommons {
         }
     }
 
+    //
+    // Function to get column entries from a file
+    //
+    public static ArrayList getColFromFile(input_file, col=0, uniqify=false, sep='\t') {
+        def vals = []
+        input_file.eachLine { line ->
+            def val = line.split(sep)[col]
+            if (uniqify) {
+                if (!vals.contains(val)) {
```

Comment:
This would make my algorithms teacher upset, better to use a `Set` in this case.  Will suggest this change elsewhere

Reply:
This will have to wait until the next release now! Minor change :)

Reply:
Yep, no problem.  There are more low hanging fruit than this for improving performance 😉